src/Compatibility/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/ApiCompatServiceProvider.cs

```diff
@@ -17,7 +17,7 @@ internal sealed class ApiCompatServiceProvider
         private readonly Lazy<ISuppressibleLog> _compatibilityLogger;
         private readonly Lazy<IApiCompatRunner> _apiCompatRunner;
 
```

Comment:
Maybe sprinkle some triple slash comments if this is public now.

Reply:
The ApiCompatServiceProvider type is internal so no need for triple slash comments. They are actually harmful as they still get embedded into the compiler generated XML file which we don't want for non public API.